server/admin.py

```diff
@@ -13,6 +17,24 @@
 )
 
 
+@admin.action(description="Export Selected")
+def export_as_csv(
+    self: admin.ModelAdmin[Membership], request: HttpRequest, queryset: QuerySet[Membership]
+) -> HttpResponse:
+    meta = self.model._meta
+    field_names = [field.name for field in meta.fields]
```

Comment:
so we'll player ID, membership ID... as the columns right ?

Reply:
we will get player name